modules/op-release-notes-1-4.adoc

```diff
@@ -0,0 +1,143 @@
+
+* You can use the `--clustertask` option in the `tkn tr delete` command to delete all the task runs associated with a particular cluster task.
+
+* Support for using Knative service with the `EventListener` resource is added by introducing a new `customResource` field.
+
+* An error message is displayed when an event payload does not use the JSON format.
+
+* The source control interceptors such as GitLab, BitBucket, and GitHub, now use the new `InterceptorRequest` or `InterceptorResponse` type interface.
+
+* A new CEL function `marshalJSON` is implemented so that you can encode a JSON object or an array to a string.
+
+* An HTTP handler for serving the CEL and the source control core interceptors is added. It packages four core interceptors into a single HTTP server that is deployed in the `tekton-pipelines` namespace. The `EventListener` object forwards events over the HTTP server to the interceptor. Each interceptor is available at a different path. For example, the CEL interceptor is available on the `/cel` path.
+
+
+[id="deprecated-features-1-4_{context}"]
+== Deprecated features
+
+* The `build-gcs` sub-type in the pipeline resource storage, and the `gcs-fetcher` image, are not supported.
+
+* In the `taskRun` field of cluster tasks, the label `tekton.dev/task` is removed.
+
+* For webhooks, the value `v1beta1` corresponding to the field `admissionReviewVersions` is removed.
+
+* The `creds-init` helper image for building and deploying is removed.
+
+* In the triggers spec and binding, the deprecated field `template.Name` is removed in favor of `template.Ref`.
+
+* For `EventListener` custom resources/objects, the fields `PodTemplate` and `ServiceType` are deprecated in favor of `Resource`.
+
+* The deprecated spec style embedded bindings is removed.
+
+* The `spec` field is removed from the `triggerSpecBinding`.
+
+* The event ID representation is changed from a five-character random string to a UUID.
+
+
+[id="known-issues-1-4_{context}"]
+== Known issues
+
+* In the **Developer** perspective, Pipeline Metrics and Triggers are available only on OpenShift Pipelines v4.7.6 or later versions.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, the `tkn hub` command is not supported.
+
+* When you run Maven and Jib Maven cluster tasks on an IBM Power Systems (ppc64le), IBM Z, and LinuxONE (s390x) clusters, set the `MAVEN_IMAGE` parameter value to `maven:3.6.3-adoptopenjdk-11`.
+
+* On IBM Power Systems, IBM Z, and LinuxONE, a Jib Maven cluster task fails.
+
+* Triggers throw error resulting from bad handling of JSON, if you have the following configuration in the trigger binding:
++
+[source, terminal]
+----
+params:
+  - name: github_json
+    value: $(body)
+----
+To resolve the issue:
+** If you are using triggers version `v0.11.0` and above, use the `marshalJSON` CEL function, which takes a JSON object or array and returns the JSON encoding of that object/array as a string.
```

Comment:
are using/use L93 and L94
drop the word 'version'. And remove the backticks on `v0.11.0`
replace / with or